src/ParMetis-4.0.3/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8...3.5)
```

Comment:
I'm left with the warning 

```
CMake Deprecation Warning at ParMetis-4.0.3/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.
```
Very similar to the old error, so maybe it would make sense to just jump this to 3.10 now. I was initially being conservative, but I don't really see anything in the ParMETIS `CMakeLists.txt` files to make me thing it would matter.

Reply:
I just tried it, and the build was successful, so updated.